src/main/webapp/app/shared/pipes/artemis-translate.pipe.ts

```diff
@@ -4,16 +4,16 @@ import { TranslatePipe, TranslateService } from '@ngx-translate/core';
 @Pipe({
     name: 'artemisTranslate',
     pure: false,
+    standalone: true,
 })
 /**
  * a simple wrapper to prevent compile errors in IntelliJ
  */
 export class ArtemisTranslatePipe implements PipeTransform, OnDestroy {
     private translatePipe: TranslatePipe;
-    constructor(
-        private translateService: TranslateService,
-        private changeDetectorRef: ChangeDetectorRef,
-    ) {
+
+    // We cannot inject() ChangeDetectorRef here
+    constructor(translateService: TranslateService, changeDetectorRef: ChangeDetectorRef) {
```

Comment:
You could improve the comment here, by explaining _why_ we cannot do that here (or maybe even find a work around to make it work?)

Reply:
Thanks for the feedback! 😄 I tried to extend the comment in this [commit](https://github.com/ls1intum/Artemis/pull/9880/commits/2925c0d5cd9b9aaf86f23beac3775e7949541e69). Unfortunately, I already tried some ways to get around it, but I could not find a good solution. 😅  But I am open for suggestions, if anyone could think of something.